install/threadfin-install.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 tteck
+# Author: tteck (tteckster)
+# License: MIT
+# https://github.com/tteck/Proxmox/raw/main/LICENSE
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y sudo
+$STD apt-get install -y mc
+$STD apt-get install -y ffmpeg
+$STD apt-get install -y vlc
```

Comment:
Threadfin supports buffering via different "backends". For convenience it includes the two supported ones.

Buffering is important to avoid stutters and lags as well as for support of "advanced" formats such as HLS (which is my use case).

Reply:
You left out `curl` It's included in all scripts.

Reply:
Done.